src/app/checkboxes/checkboxes.demo.html

```diff
@@ -61,9 +61,51 @@
                 </div>
             </div>
         </div>
+        <div class="form-group">
+            <label>Native Indeterminate Checkbox</label>
+            <div class="checkbox-inline">
+                <input type="checkbox"
+                       id="checkrads_10"
+                       [indeterminate]="nativeIndeterminateState">
+                <label for="checkrads_10">A native indeterminate checkbox.</label>
+            </div>
+        </div>
+    </section>
+</form>
+
+<form #form="ngForm" novalidate>
+    <section class="form-block">
+        <div class="form-group">
+            <label>With ngForm & ngModel</label>
+            <div class="checkbox-inline">
+                <clr-checkbox
+                        name="termsCheckbox"
+                        id="checkrads_11"
+                        [clrChecked]="termsAgreement"
+                        [ngModel]="termsAgreement">
+                    I agree to the terms and conditions.
+                </clr-checkbox>
+            </div>
+            <div class="checkbox-inline">
```

Comment:
This `.checkbox-inline` wrapper has not `input` of `type=checkbox` inside of it?